src/main/java/Duke.java

```diff
@@ -1,10 +1,85 @@
+import java.util.Scanner;
+
 public class Duke {
+
+    public static final int MAX_TASKS = 100;
+    public static String horizontal = ("____________________________________________________________\n");
+
+    public static void sayHello() {
+        System.out.println(horizontal + "Hello! I'm Duke\n" + "What can I do for you?\n" + horizontal);
+    }
+
+    public static void sayBye() {
+        System.out.println(horizontal + "Bye. Hope to see you again soon!\n" + horizontal);
+    }
+
+    public static void addTask(String line, Task[] tasks, int numberOfTasks) {
+        if (line.contains("todo")) {
+            int startOfTask = 4;
```

Comment:
```suggestion
            int startOfTask = INDEX_OF_TASK_START;
```

Perhaps you can rename these indexs to constants to avoid the Magic Numbers